docs/integrations/sources/paypal-transaction.md

```diff
@@ -61,3 +61,5 @@ Balances sync is similarly performed with default `stream_slice_period` = 1 day,
 | Version | Date       | Pull Request | Subject |
 | :------ | :--------  | :-----       | :------ |
 | 0.1.0   | 2021-06-10 | [4240](https://github.com/airbytehq/airbyte/pull/4240) | PayPal Transaction Search API |
+| 0.1.1   | 2021-08-03 | [5155](https://github.com/airbytehq/airbyte/pull/5155) | fix start_date_min limit |

```

Comment:
could you move latest version up to make it consistent with other docs ?

Reply:
fixed